agent/nmap_agent.py

```diff
@@ -105,7 +105,9 @@ def _emit_services(self, scan_results):
                 raise ValueError(f'Incorrect ip version {version}')
             for data in generators.get_services(scan_results):
                 self.emit(selector, data)
-
+                if domain_name is not None:
+                    domain_name_service = {'name': domain_name, 'port': data.get('port'), 'schema': data.get('service', False)}
```

Comment:
Why get False?

Reply:
i was using TLS as a bool. changed it now